htdocs/product/class/product.class.php

```diff
@@ -2750,7 +2750,8 @@ public function fetch($id = 0, $ref = '', $ref_ext = '', $barcode = '', $ignore_
 
 				// Load multiprices array
 				if (getDolGlobalString('PRODUIT_MULTIPRICES') && empty($ignore_price_load)) {                // prices per segment
-					for ($i = 1; $i <= $conf->global->PRODUIT_MULTIPRICES_LIMIT; $i++) {
+					$produit_multiprices_limit = getDolGlobalString('PRODUIT_MULTIPRICES_LIMIT');
```

Comment:
int ?

Reply:
I asked myself this question since it's used within a FOR loop.
But since I've only seen getDolGlobalString elsewhere in the code, I preferred to stick with this known one.
And strangely phpstan doesn't cry for it (!)